lib/Events/BeforeFederationRedirectEvent.php

```diff
@@ -2,45 +2,43 @@
 
 namespace OCA\Richdocuments\Events;
 
-
 use OCP\Files\Node;
 use Symfony\Component\EventDispatcher\Event;
 
 class BeforeFederationRedirectEvent extends Event {
 
-    /** @var Node */
-    private $node;
-    /** @var string */
-    private $relativePath;
-    /** @var string|null */
-    private $redirectUrl = null;
-    /** @var string */
-    private $remote;
-
-    public function __construct($node, $relativePath, $remote) {
-        $this->node = $node;
-        $this->relativePath = $relativePath;
-        $this->remote = $remote;
-    }
-
-    public function getRelativePath() {
-        return $this->relativePath;
-    }
-
-    public function getNode() {
-        return $this->node;
-    }
-
-    public function getRemote() {
-        return $this->remote;
-    }
-
-    public function setRedirectUrl($redirectUrl) {
-        $this->redirectUrl = $redirectUrl;
-    }
-
-    public function getRedirectUrl() {
-        return $this->redirectUrl;
-    }
-
-}
\ No newline at end of file
+	/** @var Node */
```

Comment:
We will still use `@var` annotations, or we can migrate to `Type declarations`

Reply:
We can migrate those for Nextcloud 24 or newer, yes.